packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,61 @@
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #fff);
+    border-bottom: var(--terra-action-header-border-bottom, 1px solid #1c1f21);
+    border-left: var(--terra-action-header-border-left, 1px solid #1c1f21);
+    border-right: var(--terra-action-header-border-right, 1px solid #1c1f21);
+    border-top: var(--terra-action-header-border-top, 1px solid #1c1f21);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-fill {
+    align-self: center;
+    flex: 1 1 auto;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-end {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    position: relative;
+  }
+
+  .flex-collapse {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    max-width: 40%;
+    position: relative;
+  }
+
+  .title-container {
+    height: 100%;
+    left: 0;
+    position: relative;
+    top: 0;
+    width: 100%;
+  }
+
+  .title {
```

Comment:
Do we also need line height as a themeable variable here?

Reply:
Allowing the line height to be changed would allow for the consumer to break vertical centering on the title. Do we want to allow that?

Reply:
When the consumer changes the font-size, would they need to change the line-height to maintain vertical centering on the title?

Reply:
As currently implemented no, it would stay centered.

Reply:
I'll look into this further and see if needs to be added or how to handle.